llvm/test/CodeGen/AMDGPU/mode-register-fptrunc.gfx10plus.mir

```diff
@@ -0,0 +1,70 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
The file name should rather read gfx11plus?

Reply:
renamed